Calculator/Calculator/ExpressionParser.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by redmango1446 on 2023/06/06.
+//
+
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        var formula = Formula()
+        let inputStrings = componentsByOperators(from: input)
+        
+        let operatorCharacter = inputStrings.compactMap{
+            if $0.count == 1 {
+                return Operator(rawValue: Character($0))
+            }
+            return nil
+        }
+        
+        inputStrings.compactMap{ Double($0) }.forEach{ formula.operands.enqueue($0) }
+        operatorCharacter.forEach{ formula.operatos.enqueue($0) }
```

Comment:
다른곳에선 고차함수를 chaining을 통해 사용하고 있습니다.
하지만 이 부분은 따로 해주고있네요.
```suggestion
       inputStrings
                .compactMap{//...}
                .forEach{ formula.operatos.enqueue($0) }
```
이런 방법으로 수정해볼 수 있겠네요.